enterprise_access/apps/subsidy_request/models.py

```diff
@@ -112,6 +116,10 @@ def bulk_update(cls, subsidy_requests, field_names, batch_size=SUBSIDY_REQUEST_B
 
     class Meta:
         abstract = True
+        index_together = [
+            ['uuid', 'state'],
```

Comment:
In support of this query: https://github.com/edx/enterprise-access/blob/8573d80cb21f761699652953f1c624703ed93ebb/enterprise_access/apps/api/tasks.py#L141